src/bundle/Resources/views/themes/admin/content/tab/versions/table.html.twig

```diff
@@ -6,148 +6,150 @@
 {% set custom_actions = custom_actions|default(false) %}
 {% set have_actions = (is_draft or is_archived) or custom_actions %}
 
-<table class="ibexa-table table ibexa-table__draft-conflict {{ (is_draft and have_to_paginate ? 'mb-3' : '') }}">
-    <thead>
-        <tr class="ibexa-table__head-row">
-            {% if form is defined %}
+<div class="ibexa-scrollable-wrapper">
+    <table class="ibexa-table table ibexa-table__draft-conflict {{ (is_draft and have_to_paginate ? 'mb-3' : '') }}">
```

Comment:
you don't have an `ibexa-table` parent so i think it should either be `ibexa-draft-conflict` or `ibexa-table--draft-conflict`